presto-hive/src/test/java/com/facebook/presto/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -871,6 +871,32 @@ public void testCastNullToColumnTypes()
         assertUpdate("DROP TABLE " + tableName);
     }
 
+    @Test
+    public void testCreateEmptyBucketedPartition()
+    {
+        testCreateEmptyBucketedPartition(HiveStorageFormat.RCBINARY);
+    }
+
+    public void testCreateEmptyBucketedPartition(HiveStorageFormat storageFormat)
+    {
+        String tableName = "test_insert_partitioned_bucketed_table";
+        createPartitionedBucketedTable(tableName, storageFormat);
+
+        ImmutableList<String> orderStatusList = ImmutableList.of("F", "O", "P");
+        for (int i = 0; i < orderStatusList.size(); i++) {
+            String sql = format("CALL system.create_empty_partition('%s', '%s', ARRAY['orderstatus'], ARRAY['%s'])", TPCH_SCHEMA, tableName, orderStatusList.get(i));
+            getQueryRunner().execute(getSession(), sql);
+            assertQuery(
+                    "SELECT count(*) FROM \"test_insert_partitioned_bucketed_table$partitions\"",
+                    "SELECT " + (i + 1));
+
+            assertQueryFails(sql, "Partition already exists.*");
+        }
+
+        assertUpdate("DROP TABLE " + tableName);
+        assertFalse(getQueryRunner().tableExists(getSession(), tableName));
```

Comment:
We don't need this check here, as the purpose of this test is not to validate `DROP TABLE` behavior.

Reply:
@electrum : The code is refactored from the original tests. I think it's used as a "post-condition" assertion to make sure the table is really cleaned up and to make sure different tests are well separated. 

I can see your point that the tests doesn't really check the `DROP TABLE`. And I am OK either way :) .